dev/core/src/com/google/gwt/core/linker/SingleScriptTemplate.js

```diff
@@ -119,6 +104,7 @@ function __MODULE_FUNC__() {
       return (slashIndex >= 0) ? path.substring(0, slashIndex + 1) : '';
     };
 
+    var thisScript = $doc.currentScript;
```

Comment:
According to caniuse.com `currentScript` is not supported by IE 11. So you can not use it for now unless a decision has been made to drop IE 11 in the next GWT release.

Reply:
Are you OK to make the code try currentScript and if fails to try the old implementation?

Reply:
That seems reasonable to me - put an explicit note so that when IE 11 is finally truly dead we know that this can be simplified out?

Reply:
> Are you OK to make the code try currentScript and if fails to try the old implementation?

Yes that would be fine.

Reply:
@Mivr do you want to make this change so we can see about merging?

Reply:
@niloc132 Yes, sorry for the delay I am just way too busy and warning are not the highest priority

Reply:
Yep no problem, just checking in with various open PRs to see what we can land before trying to do a release. No rush, we've got other pending PRs, but this one is small enough we could ship without it if you don't get the chance to get to it.